src/sql/workbench/contrib/queryplan2/browser/queryPlan.ts

```diff
@@ -0,0 +1,738 @@
+		// new Operation("ConstTableGet",          SR.Keys.ConstTableGet,          SR.Keys.ConstTableGetDescription,           "Const_table_get_32x.ico"),
+		// new Operation("ControlToComputeNodes",  SR.Keys.ControlToComputeNodes,  SR.Keys.ControlToComputeNodesDescription,   "Control_to_compute_32x.ico"),
+		// new Operation("ExternalBroadcast",      SR.Keys.ExternalBroadcast,      SR.Keys.ExternalBroadcastDescription,       "External_broadcast_32x.ico"),
+		// new Operation("ExternalExport",         SR.Keys.ExternalExport,         SR.Keys.ExternalExportDescription,          "External_export_32x.ico"),
+		// new Operation("ExternalLocalStreaming", SR.Keys.ExternalLocalStreaming, SR.Keys.ExternalLocalStreamingDescription,  "External_local_streaming_32x.ico"),
+		// new Operation("ExternalRoundRobin",     SR.Keys.ExternalRoundRobin,     SR.Keys.ExternalRoundRobinDescription,      "External_round_robin_32x.ico"),
+		// new Operation("ExternalShuffle",        SR.Keys.ExternalShuffle,        SR.Keys.ExternalShuffleDescription,         "External_shuffle_32x.ico"),
+		// new Operation("Get",                    SR.Keys.Get,                    SR.Keys.GetDescription,                     "Get_32x.ico"),
+		// new Operation("GbApply",                SR.Keys.GbApply,                SR.Keys.GbApplyDescription,                 "Apply_32x.ico"),
+		// new Operation("GbAgg",                  SR.Keys.GbAgg,                  SR.Keys.GbAggDescription,                   "Group_by_aggregate_32x.ico"),
+		// new Operation("Join",                   SR.Keys.Join,                   SR.Keys.JoinDescription,                    "Join_32x.ico"),
+		// new Operation("LocalCube",              SR.Keys.LocalCube,              SR.Keys.LocalCubeDescription,               "Intrinsic_32x.ico"),
+		// new Operation("Project",                SR.Keys.Project,                SR.Keys.ProjectDescription,                 "Project_32x.ico"),
+		// new Operation("Shuffle",                SR.Keys.Shuffle,                SR.Keys.ShuffleDescription,                 "Shuffle_32x.ico"),
+		// new Operation("SingleSourceRoundRobin", SR.Keys.SingleSourceRoundRobin, SR.Keys.SingleSourceRoundRobinDescription,  "Single_source_round_robin_32x.ico"),
+		// new Operation("SingleSourceShuffle",    SR.Keys.SingleSourceShuffle,    SR.Keys.SingleSourceShuffleDescription,     "Single_source_shuffle_32x.ico"),
+		// new Operation("Trim",                   SR.Keys.Trim,                   SR.Keys.TrimDescription,                    "Trim_32x.ico"),
+		// new Operation("Union",                  SR.Keys.Union,                  SR.Keys.UnionDescription,                   "Union_32x.ico"),
+		// new Operation("UnionAll",               SR.Keys.UnionAll,               SR.Keys.UnionAllDescription,                "Union_all_32x.ico"),
+
+
+		this._iconMap['Adaptive_Join_32x.ico'] = 'adaptiveJoin';
```

Comment:
this code block can away too once we get the STS fix to use the more generic icon names.